source/common/init/manager_impl.h

```diff
@@ -35,27 +37,42 @@ class ManagerImpl : public Manager, Logger::Loggable<Logger::Id::init> {
   void add(const Target& target) override;
   void initialize(const Watcher& watcher) override;
 
+  // Expose the const reference of target_names_count_ hash map to public.
+  const absl::flat_hash_map<std::string, uint32_t>& unreadyTargets() const;
+
 private:
+  // Callback function without parameters, decrease unready targets count by 1.
   void onTargetReady();
+
+  // Callback function with an additional target_name parameter, decrease unready targets count by
+  // 1, update target_names_count_ hash map.
+  void onTargetReadySendTargetName(absl::string_view target_name);
```

Comment:
We can name this `onTargetReady(absl::string_view target_name);`

Reply:
Updated.